ush/soca/run_jjobs.py

```diff
@@ -112,6 +112,7 @@ def setupexpt(self):
             "start": "warm",
             "gfs_cyc": "0",
             "resdet": self.config['resdet'],
+            "resens": self.config['resens'],
```

Comment:
Good to add the `resens` line back in ... we just need to be mindful that C48 is the lowest resolution supported by g-w.   Previously we had C24 for the ensemble resolution.   This caused g-w scripts to abort.    Since `test/soca/gw/run_jjobs.yaml.test` now specifies C48 for `resens`, we're OK.